scripts/main.py

```diff
@@ -0,0 +1,150 @@
+import os
+from dotenv import load_dotenv
+import tweepy as tw
+import re
+import string
+import nltk
+import pandas as pd
+from sklearn.cluster import DBSCAN
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+# NLTK variables to aid in cleaning/preprocessing
+ps = nltk.PorterStemmer()
+
+# Pandas dataframe for tweets
+df = pd.DataFrame(columns=["id", "tweet", "popularity"])
+
+
+class Results:
+    def __init__(self):
+        """Initialize final results of the analysis"""
+        self.clusters_count = df.clusters.value_counts()
+        self.df_results = df.groupby(["clusters"]).max().reset_index()
+        print("Number of tweets per cluster: \n{}".format(self.clusters_count))
+        print("Top cluster tweets: \n{}".format(self.df_results.to_string()))
+
+
+def authorise_api():
```

Comment:
This piece of code is not required here, 

let me know what are the pieces of information you require to get from the `tweepy`, 

this will be taken care in a separate file to make this modular

Reply:
Well for starters, I need the trending tweet at any location. Then from that trending topic, I will be using the `tweepy.Cursor()` to fetch tweets with the trending topic as a query.

Reply:
Sure just define your function with tweets as a parameter and I will send the tweets from a different file.